index.html

```diff
@@ -0,0 +1,570 @@
+            </ul>  
+              <ul class="sub-list-base">
+                <li class="listings">Email Newsletters</li>
+                <li class="listings">Corporate Subscriptions</li>
+                <li class="edu">Education Rate</li>
+                <li class="listings">Mobile Applications</li>
+                <li class="listings">Replica Edition</li>
+                <li class="listings">International</li>
+                <li class="listings">Cananda</li>
+                <li class="listings">Espanol</li>
+                <li class="listings">中文网</li>
+              </ul>
+          </li>  
+        </ul>
+        </div>
+      </div>
+      <div class="bottom-list">
+        <div class="footer-text-1">
+          <a class="footer-text" href="">&copy; 2021 The New York Times Company</a>
+        </div>
+        <ul class="footer-down">
+          <li class="footer-items"><a class="footer-list" href="">NYTco</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Accessibilty</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Contact Us</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Work with us</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Advertise</a></li>
+          <li class="footer-items"><a class="footer-list" href="">T Brand Studio</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Your Ad Choices</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Privacy Policy</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Terms of Service</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Terms of Sale</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Site Map</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Canada</a></li>
+          <li class="footer-items"><a class="footer-list" href="">International</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Help</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Subscriptions</a></li>
```

Comment:
- [ ] Kindly use a valid value e.g. "#"  for the href attributes

Reply:
> * [x]  Kindly use a valid value e.g. "#"  for the href attributes

Thank you for your review, I have implemented all the changes, I will submit for a re-review.